launch-sublime-platform.sh

```diff
@@ -1,18 +1,53 @@
 #!/bin/bash
 
-if [ -z "$auto_updates" ]; then
+. ./utils.sh
+
+if [ -z "$interactive" ]; then
+    interactive="true"
+fi
+
+if [ "$interactive" != "true" ] && [ -z "$auto_updates" ]; then
     auto_updates=true
 fi
 
 if ! ./preflight_checks.sh; then
     exit 1
 fi
 
-# If this command is modified we might need a more sophisticated check below (worse case is more updates than intended)
-update_command="cd ""$(pwd)"" && bash -lc ./update-and-run.sh"
+print_info "Configuring automatic updates..."
+if [ "$interactive" == "true" ] && [ -z "$auto_updates" ]; then
+    while true; do
+        # Since this script is intended to be piped into bash, we need to explicitly read input from /dev/tty because stdin
+        # is streaming the script itself
+        read -rp "Would you like to enable auto-updates? If yes, your terminal may request permissions to make updates to your computer: [Y/n]: " yn </dev/tty
+        case $yn in
+            [Yy]* | "" ) auto_updates="true"; break;;
```

Comment:
The only difference from before is that I added the `| ""` to make the default be true.